### ISSUE TITLE
Radio button keeps gray circle inside after toggling disable->enable

### ISSUE THREAD

**Issue by mfonville** [google/material-design-lite#4212]
MDL 1.1.3
Reproducible in Chrome, not in Firefox (that correctly redraws):

go to http://opengapps.org/
select arm -> 5.0 -> stock
You will see that 'super' is disabled
select 5.1 -> super
You will see that 'super' has a gray circle inside instead of the teal.

**Comment by Garbee** [comment]
What version of Chrome are you running? Works for me in Version 51.0.2679.0 dev (64-bit).

**Comment by Zodiase** [comment]
![image](https://cloud.githubusercontent.com/assets/1815174/13888896/d23be71a-ed11-11e5-95f3-07a847b1954b.png)

No problem in 49.0.2623.87 (64-bit)

**Comment by mfonville** [comment]
Linux Version 49.0.2623.87 (64-bit)
![grayradio](https://cloud.githubusercontent.com/assets/6221544/13889336/61b7632c-ed46-11e5-8ff6-efa4ac3d0089.png)

It was confirmed by @sigv ; what is your configuration?

**Comment by Zodiase** [comment]
Mine is OS X 10.11.3 (15D21)

**Comment by sigv** [comment]
Yes, I am seeing the same behaviour as @mfonville attached in the picture running Version 49.0.2623.87 (64-bit) on Ubuntu 15.10 (Linux x86_64).

Follow the steps:
1. Select ARM -> 5.1 -> Super
2. Select ARM -> 5.0 (the super button becomes disabled)
3. Select ARM -> 5.0 -> Stock
4. Select ARM -> 5.1 -> Super

**Comment by Garbee** [comment]
Ok, now it is replicating in my Chrome.

**Comment by sigv** [comment]
The issue does not appear to be there with MDL 1.3.0 and Chrome 57.0.2987.110.
Someone with rights should probably just close this issue.
